File: Algorithms/Warmup/Compare the Triplets/comparethetriplets.py
a_scores = input().strip().split(" ")
b_scores = input().strip().split(" ")

alice = 0
bob = 0

for i in range(len(a_scores)):
    if int(a_scores[i]) == int(b_scores[i]):
        continue
    elif int(a_scores[i]) > int(b_scores[i]):
        alice += 1
    else:
        bob += 1

print(alice, bob)

File: Learn/Algorithms/Warmup/Compare the Triplets/comparethetriplets.cpp
#include <iostream>

using namespace std;
int main() {
    int scores[2][3];

    for (int i = 0; i < 2; i++) {
        for (int j = 0; j < 3; j++) {
            cin >> scores[i][j];
        }
    }

    int alice = 0;
    int bob = 0;
    for (int i = 0; i < 3; i++) {
        if (scores[0][i] == scores[1][i]) {
            continue;
        } else if (scores[0][i] > scores[1][i]) {
            alice++;
        } else {
            bob++;
        }
    }

    cout << alice << " " << bob << endl;

    return 0;
}
